src/po/template.pot

```diff
@@ -1,12 +1,21 @@
 msgid ""
 msgstr ""
 "Language: <<specify ISO language string here>>\n"
-"Project-Id-Version: 2015-11-04T09:07:28.962Z\n"
+"Project-Id-Version: 2016-01-12T11:14:13.921Z\n"
```

Comment:
This file is automatically generated by grunt. To keep the commits clean and easier to review, I would recommend to keep this file out of commits. It can be updated in specific commits.